usaspending_api/awards/tests/unit/test_subawards.py

```diff
@@ -49,13 +49,22 @@ def test_all_subawards(mock_matviews_qs):
     subawards_logic = svs._business_logic(test_params)
     assert [] == subawards_logic
 
+    sort("id",[strip_award_id(subaward_3), strip_award_id(subaward_1), strip_award_id(subaward_2)])
+    sort("amount", [strip_award_id(subaward_3), strip_award_id(subaward_2), strip_award_id(subaward_1)])
+    sort("action_date", [strip_award_id(subaward_2), strip_award_id(subaward_1), strip_award_id(subaward_3)])
+    sort("recipient_name", [strip_award_id(subaward_2), strip_award_id(subaward_3), strip_award_id(subaward_1)])
+
+def sort(sort,response):
```

Comment:
Good idea to create a function to test numerous sort values. Please rename `sort()` to be more descriptive that it is testing sort behavior

Reply:
Also please change `response` to `expected_response`